plugins/outputs/bigquery/bigquery.go

```diff
@@ -111,6 +130,20 @@ func (s *BigQuery) Write(metrics []telegraf.Metric) error {
 	return nil
 }
 
+func (s *BigQuery) writeCompact(metrics []telegraf.Metric) error {
+	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(ctx, time.Duration(s.Timeout))
+	defer cancel()
+
+	inserter := s.client.DatasetInProject(s.Project, s.Dataset).Table(s.CompactTable).Inserter()
```

Comment:
Could you please add at least a comment here that this call does not panic even though the table might not exist and the `inserter` will not be `nil` in that case!?!